src/components/SingleArticleCard.jsx

```diff
@@ -12,7 +41,10 @@ const SingleArticleCard = ({ presentArticle }) => {
       <p className="article-date">Posted at {formattedDate}</p>
       <article className="article-body">{body}</article>
       <img className="article-image" src={article_img_url} alt={title} />
-      <p className="article-votes">Votes: {votes}</p>
+      <button className="vote-button" onClick={() => changeVotes(1)}>+</button>
```

Comment:
CSS of these buttons needs fixing, whilst styling can be completed later, buttons being in random places needs to be fixed.

The colours are also not very accessible at all, definitely chose some better font colours to contrast the background.

The buttons work and we do get an error message if there is an error, however I would say the downvote button being highlighted after an upvote suggests it has worked. I think it may be beneficial to provide some user feedback to inform them that they cannot do this.